docs/en/guides/containers/oss/eks/self-managed-to-managed-oss.md

```diff
@@ -0,0 +1,204 @@
+# EKS Observability : Managed Observability Services to AWS Managed Open Source Observability Services
+In the ever-evolving world of modern software development, observability has become a critical aspect of ensuring the reliability, performance, and scalability of applications. While self-managed tools like Prometheus and Datadog have been invaluable in providing insights into the health and behavior of systems, the increasing complexity and scale of modern architectures often demand more robust and scalable solutions. Enter managed observability services, such as Amazon Managed Prometheus and Amazon Managed Grafana. These fully managed offerings from AWS aim to simplify the management and operation of observability tools, freeing up valuable time and resources for organizations to focus on their core business objectives.
+
+In this guide, we will explore the journey of migrating from self-managed observability tools to Amazon's managed services.
+
+# Why Managed Observability Services ?
+By leveraging managed services, organizations can benefit from:
+
+1. Scalability and Elasticity: Amazon's managed services are designed to scale seamlessly, allowing you to accommodate fluctuating workloads and handle spikes in data ingestion without compromising performance.
+
+2. Reduced Operational Overhead: With managed services, AWS takes care of the heavy lifting – patching, upgrading, and maintaining the underlying infrastructure, freeing up your team to concentrate on higher-value tasks.
+
+3. Enhanced Security and Compliance: AWS's managed services adhere to stringent security and compliance standards, providing peace of mind and helping organizations meet regulatory requirements.
+
+4. Seamless Integration: Amazon's managed observability services seamlessly integrate with other AWS services and third-party tools, enabling a cohesive and unified monitoring experience across your entire infrastructure.
+
+# Uninstall Self Managed Prometheus
+If you have deployed Prometheus using a deployment manifest, you can delete all the resources by running the following command:
+```
+kubectl delete -f <path/to/prometheus-deployment-manifest.yaml>
+```
+If you have used Helm to install Prometheus, you can uninstall it with the following command:
+```
+helm uninstall <prometheus-release-name>
+```
+If you have created any ConfigMaps or Secrets for Prometheus configuration, you can delete them as well:
+```
+kubectl delete configmap <prometheus-config-map-name>
+kubectl delete secret <prometheus-secret-name>
+```
+
+# Uninstall DataDog
+
+If you have DataDog installed, use the following commands to uninstall DataDog
+
+```
+helm uninstall datadog-agent
+helm delete datadog-operator
+```
+If you have a secret created for DataDog, then Use the following command to delete the DataDog Secret
+```
+kubectl delete secret <datadog-agent-secret-name> -n <namespace>
+```
+
+# Setup Amazon Managed Service for Prometheus
```

Comment:
What about the Accelerators?

Reply:
I have added separate section explaining how accelerators can be used.